docs/content/Deployment/Platform-Serverless-AWS.md

```diff
@@ -0,0 +1,199 @@
+---
+title: Deploy with Serverless Framework on AWS
+permalink: /deployment/platforms/serverless/aws
+category: Deployment
+subCategory: Platforms
+menuOrder: 2
+---
+
+This guide walks you through deploying Cube.js with the [Serverless
+Framework][link-sls] on [AWS][link-aws].
+
+<!-- prettier-ignore-start -->
+[[warning |]]
+| This is an example of a production-ready deployment, but real-world
+| deployments can vary significantly depending on desired performance
+| and scale.
+<!-- prettier-ignore-end -->
+
+## Prerequisites
+
+- An AWS account
+- An [Elasticache Redis][aws-redis] cluster URL
+- A separate Cube Store or external database
+- Node.js 10+
+- Serverless Framework
+
+## Configuration
+
+Create a Serverless Framework project by creating a `serverless.yml`. A
+production-ready stack would at minimum consist of:
+
+- One or more Cube.js API instance
+- A Cube.js Refresh Worker
+
+The `serverless.yml` for an example project is provided below:
+
+```yaml
+service: hello-cube-sls
+
+provider:
+  name: aws
+  runtime: nodejs12.x
+  iamRoleStatements:
+    - Effect: 'Allow'
+      Action:
+        - 'sns:*'
+      Resource: '*'
+  environment:
+    CUBEJS_DB_HOST: <YOUR_DB_HOST_HERE>
+    CUBEJS_DB_NAME: <YOUR_DB_NAME_HERE>
+    CUBEJS_DB_USER: <YOUR_DB_USER_HERE>
+    CUBEJS_DB_PASS: <YOUR_DB_PASS_HERE>
+    CUBEJS_DB_PORT: <YOUR_DB_PORT_HERE>
+    CUBEJS_REDIS_URL: <YOUR_REDIS_URL_HERE>
+    CUBEJS_DB_TYPE: <YOUR_DB_TYPE_HERE>
+    CUBEJS_API_SECRET: <YOUR_API_SECRET_HERE>
+    CUBEJS_APP: '${self:service.name}-${self:provider.stage}'
+    NODE_ENV: production
```

Comment:
@hassankhan do we still depend on the NODE_ENV?

Reply:
No, but Serverless Framework uses it internally for packaging the Lambda `.zip` file